src/dbos-runtime/config.ts

```diff
@@ -481,12 +480,18 @@ export function overwrite_config(
 
   const poolConfig = constructPoolConfig(configFile!);
 
-  const OTLPExporterConfig: OTLPExporterConfig = providedDBOSConfig.telemetry?.OTLPExporter || {};
+  if (!providedDBOSConfig.telemetry.OTLPExporter) {
+    providedDBOSConfig.telemetry.OTLPExporter = {};
+  }
   if (configFile!.telemetry?.OTLPExporter?.tracesEndpoint) {
-    OTLPExporterConfig.tracesEndpoint = configFile!.telemetry.OTLPExporter.tracesEndpoint;
+    providedDBOSConfig.telemetry.OTLPExporter.tracesEndpoint = configFile!.telemetry.OTLPExporter.tracesEndpoint;
+  }
+  if (configFile!.telemetry?.OTLPExporter?.logsEndpoint) {
+    providedDBOSConfig.telemetry.OTLPExporter.logsEndpoint = configFile!.telemetry.OTLPExporter.logsEndpoint;
   }
 
   const overwritenDBOSConfig = {
+    ...providedDBOSConfig,
```

Comment:
What does this do?

Reply:
merge whatever was given with the overwrites